llarp/service/endpoint.cpp

```diff
@@ -802,7 +802,7 @@ namespace llarp
     void
     Endpoint::LookupNameAsync(
         std::string name,
-        std::function<void(std::optional<std::variant<Address, RouterID>>)> handler)
+        std::function<void(std::optional<var::variant<Address, RouterID>>)> handler)
```

Comment:
This should stay `std::variant`: the `var` function provides just `var::get` and `var::visit` which work on `std::variant` which mac *does* support as long as you don't use `std::get` or `std::visit` on them.

Reply:
fixed